tdrs-backend/tdpservice/data_files/admin/admin.py

```diff
@@ -23,11 +23,44 @@ def has_change_permission(self, request, obj=None):
         """Read only permissions."""
         return False
 
-
 @admin.register(DataFile)
 class DataFileAdmin(ReadOnlyAdminMixin, admin.ModelAdmin):
     """Admin class for DataFile models."""
 
+    class Media:
+        """Media class for DataFileAdmin."""
+
+        js = ('admin/js/admin/mymodel.js',)
+
+    actions = ['reparse_cmd']
+
+    def reparse_cmd(self, request, queryset):
+        """Reparse the selected data files."""
+        files = queryset.values_list("id", flat=True)
+        file_ids = ",".join(map(str, files))
+        from django.core.management import call_command
+        from django.utils.translation import ngettext
+        from django.contrib import messages
+        call_command("clean_and_reparse", f"-f {file_ids}")
+        self.message_user(
+            request,
+            ngettext(
+                "%d file successfully submitted for reparsing.",
+                "%d files successfully submitted for reparsing.",
+                files.count(),
+            )
+            % files.count(),
+            messages.SUCCESS,
+        )
+
+    def get_actions(self, request):
```

Comment:
Note: if not admin, then the reparse command is not shown

Reply:
evidence below that digit cant reparse or view reparse model cc: @ttran-hub 

![digit_cant_reparse](https://github.com/user-attachments/assets/71d24ee9-a514-4ace-bedf-6a2439f629a1)